app/models/agents/trigger_agent.rb

```diff
@@ -13,7 +13,7 @@ class TriggerAgent < Agent
 
       The `value` can be a single value or an array of values. In the case of an array, if one or more values match then the rule matches. 
 
-      All rules must match for the Agent to match.  The resulting Event will have a payload message of `message`.  You can use liquid templating in the `message, have a look at the [Wiki](https://github.com/cantino/huginn/wiki/Formatting-Events-using-Liquid) for details.
+      All rules must match for the Agent to match.  The resulting Event will have a payload message of `message`, plus any additional keys not one of the TriggerAgent config keys (`rules`, `keep_event`, `expected_receive_period_in_days`).  You can use liquid templating in the `message` and other keys; have a look at the [Wiki](https://github.com/cantino/huginn/wiki/Formatting-Events-using-Liquid) for details.
```

Comment:
I might prefer putting extra key/value pairs into a nested `extra_fields` hash.  What do you think?